### PR TITLE
Updates: BYO examples to remove docker restart

### PR DESCRIPTION
**Pull request by djarpin**
Tested both notebooks and build_and_push.sh in fresh SageMaker Notebook Instance and both worked.

### CODE REVIEW

File: advanced_functionality/scikit_bring_your_own/container/build_and_push.sh [hunk 1]
```diff
@@ -13,6 +13,9 @@ then
     exit 1
 fi
 
+chmod +x decision_trees/train
+chmod +x decision_trees/serve
+
```
Comment (mckev-amazon) [inline advanced_functionality/scikit_bring_your_own/container/build_and_push.sh]: Is this a related change?

Comment (djarpin) [reply advanced_functionality/scikit_bring_your_own/container/build_and_push.sh]: I snuck that one in there.  It was a point of misalignment between the notebook's shell command and the build_and_push.sh that was called out in this [Issue](https://github.com/awslabs/amazon-sagemaker-examples/issues/179).

Comment (mckev-amazon) [reply advanced_functionality/scikit_bring_your_own/container/build_and_push.sh]: Sounds good!  Not an issue, from my perspective, as long as both issues are referenced by the commit (which you've done).
